src/main/java/net/sf/jabref/logic/xmp/XMPUtil.java

```diff
@@ -626,11 +626,10 @@ private static void writeToDCSchema(XMPSchemaDublinCore dcSchema,
              */
             if ("author".equals(field)) {
                 String authors = resolvedEntry.getField(field);
-                AuthorList list = AuthorList.getAuthorList(authors);
+                AuthorList list = AuthorList.getAuthors(authors);
 
-                int n = list.size();
-                for (int i = 0; i < n; i++) {
-                    dcSchema.addCreator(list.getAuthor(i).getFirstLast(false));
```

Comment:
previously it was AuthorList.getAuthorList.getAuthorList...has also a duplication. Mhh...what do you think about renaming the first getAuthor to parse? I.e `AuthorList.parse(String).getAuthors`. That the parsing is actually cached shouldn't be a worry for the name.